website/docs/provider_reference.html.markdown

```diff
@@ -0,0 +1,328 @@
+---
```

Comment:
Not sure how this extra file ended up in here, but it can probably be removed from the commit